terraform/environments/ops/main.tf

```diff
@@ -6,3 +6,13 @@ module "emblem_ops" {
   repo_owner      = var.repo_owner
   repo_name       = var.repo_name
 }
+
+module "emblem_website_e2e_test" {
```

Comment:
let's shorten this to `website_e2e_test`

Reply:
Done. 👍